src/hooks/useMember/useMember.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useCallback, useContext, useEffect, useState } from 'react'
+import {
+  getMember as getMemberFn,
+  setMember as setMemberFn,
+  setActiveChannel as setActiveChannelFn,
+  removeMember as removeMemberFn,
+} from './utils'
+
+export type ActiveMember = {
```

Comment:
Just came to me - we should also keep the active account address, we will need that to sign the transactions (you can have a number of accounts in your extension) and we need to pick one that we want the user to sign with. This also creates a question - what should be a structure of this. The hierarchy is the following `account -> member -> channel` so if we want to include an address as well, the current structure may need to change. What do you think? Sorry, I haven't thought of this before